build/relsignDSCModules.sh

```diff
@@ -0,0 +1,309 @@
+#!/bin/bash
+
+# Script to sign DSC modules and drop off signed packages (in zip format) into build share.
+# It generates a checksum file for each DSC module, uploads to azure storage account using SAS_SOURCE_URI,
+# then downloads the signed *.asc file from SAS_DEST_URI and 
+# serializes it into the zip package for given DSC module.
```

Comment:
Definitely, you need usage here. It should be trivial to look at a script to understand usage, or run a script with `-h` or `-?` to get usage.

Does this script replace the existing signing script? If not, when is this used vs. the existing signing script? If this replaces the existing script, then shouldn't you remove that as part of this pull request?